Gemfile

```
@@ -1,12 +1,12 @@
 source "http://rubygems.org"
 
-gem 'rake', '~> 10'
-gem 'rspec-collection_matchers', '~> 1.1'
-gem 'rdoc', '~> 4.2'
+gem 'rake', '~> 13'
+gem 'rspec-collection_matchers', '~> 1.2'
+gem 'rdoc', '~> 6.2.1'
 gem 'simplecov', :platforms => :mri_19
 
 group :multipart_support do
-  gem 'multipart-post', '~> 2'
+  gem 'multipart-post'
```

we need this fixed as I don't think wrest works with 1.x of this gem. I'd suggest:
```suggestion
  gem 'multipart-post',  '>= 2.0', '< 3.0'
```